apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/message/ReleaseMessageScanner.java

```diff
@@ -59,10 +66,12 @@ public ReleaseMessageScanner() {
   public void afterPropertiesSet() throws Exception {
     databaseScanInterval = bizConfig.releaseMessageScanIntervalInMilli();
     maxIdScanned = loadLargestMessageId();
+    missingReleaseMessages = Maps.newHashMap();
     executorService.scheduleWithFixedDelay(() -> {
       Transaction transaction = Tracer.newTransaction("Apollo.ReleaseMessageScanner", "scanMessage");
       try {
         scanMessages();
+        scanMissingMessages();
```

Comment:
Is it necessary to scan missing message first here?

i.e change to
```java
        scanMissingMessages();
        scanMessages();
```

Reply:
I think it's reasonable to scan missing message first.